bowtie/_report.py

```diff
@@ -314,8 +314,20 @@ def flat_results(self):
     def generate_badges(self, target_dir: Path, dialect: str):
         label = _DIALECT_URI_TO_SHORTNAME[dialect]
         for impl in self.implementations:
+            dialectVersions = impl["dialects"]
             if dialect not in impl["dialects"]:
```

Comment:
I assume creating a variable was in order to be able to reuse it, so you should do that here

```suggestion
            if dialect not in dialect_versions:
```